File: cpubound/pascal.go
package main

import (
       "fmt"
)

func main() {
     m := 10000
     prev := []int{1}
     //fmt.Println(prev)
     for i := 0; i < m; i++ {
     	 curr := []int{1}
	 for j := 1; j < i; j++ {
	     curr = append(curr, prev[j - 1] + prev[j])
	 }
	 curr = append(curr, 1)
	 //fmt.Println(curr)
	 prev = curr
    }
    fmt.Println("Done")
}

File: cpubound/pascal.py
import sys
#from cdecimal import *
from decimal import *
c = getcontext()
#c.prec = MAX_PREC

m = int(sys.argv[1])

#one = 1.
one = 1
s = 0


prev = [1]
print(prev)
for row in xrange(1, m):
    curr = [1]
    for index in xrange(1, row):
        s += 1
        curr.append(prev[index-1] + prev[index])
    curr.append(1)
    #print curr
    prev = curr


print s           